src/contrib/cluster/Akka.Cluster.Tools.Tests/Akka.Cluster.Tools.Tests.csproj

```diff
@@ -13,6 +13,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="$(TestSdkVersion)" />
+    <PackageReference Include="System.ValueTuple" Version="4.4.0" />
```

Comment:
Looks like this is what's causing the issues on Linux - let me take a look and see if I can figure out why.